packages/pd-manager-rest-api/src/types.ts

```diff
@@ -0,0 +1,19 @@
+import { GenericAuthArgs, ISingleEndpointOpts } from '@sphereon/ssi-express-support'
+import { IPDManager } from '@sphereon/ssi-sdk.pd-manager'
+import { IAgentContext, IDataStore, IDIDManager, IKeyManager } from '@veramo/core'
+
+export type PDManagerMRestApiFeatures = 'pd_read' | 'pd_write' | 'pd_delete'
+
+export interface IPDManagerAPIEndpointOpts {
+  endpointOpts?: {
+    basePath?: string
+    globalAuth?: GenericAuthArgs & { securePDManagerEndpoints?: boolean }
+    pdRead?: ISingleEndpointOpts // Overrides read pd entity path
+    pdWrite?: ISingleEndpointOpts // Overrides write pd entity path
+    pdDelete?: ISingleEndpointOpts // Overrides write pd entity path
+  }
+  enableFeatures?: PDManagerMRestApiFeatures[]
+}
+
+export type IRequiredPlugins = IPDManager & IDataStore & IKeyManager & IDIDManager
```

Comment:
why are so many plugins required?

Reply:
Copy from contact manager api.
I do not know what depends on IKeyManager & IDIDManager. They are everywhere in other places so they are here also.
Can they go out?